### PR TITLE
JAMES-2057 Provide missing javadocs

### PR DESCRIPTION
**Pull request by chibenwa**
Before:

11_before

After:

11_after

### CODE REVIEW

File: server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/matchers/Not.java [hunk 1]
```diff
@@ -31,17 +31,17 @@
 import com.google.common.base.Optional;
 import com.google.common.collect.Lists;
 
+/**
+ * This is the Not CompositeMatcher - consider what wasn't in the result set
+ * of each child matcher. Of course it is easier to understand if it only
+ * includes one matcher in the composition, the normal recommended use. @See
+ * CompositeMatcher interface.
+ *
+ * @return Collectiom of Recipient from the Negated composition of the child
```
Comment (quynhn) [inline server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/matchers/Not.java]: s/Recipient/Recipients

File: server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/matchers/Or.java [hunk 2]
```diff
@@ -28,15 +28,15 @@
 import org.apache.mailet.MailAddress;
 import org.apache.mailet.Matcher;
 
+/**
+ * This is the Or CompositeMatcher - consider it to be a union of the
+ * results.
+ *
+ * @return Collection of Recipient from the Or composition results of the
```
Comment (quynhn) [inline server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/matchers/Or.java]: s/Recipient/Recipients/

File: server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/matchers/Not.java (hunk 1 above)
Comment (chibenwa) [inline server/mailet/mailetcontainer-camel/src/main/java/org/apache/james/mailetcontainer/impl/matchers/Not.java]: Ahaha these are not written by me, it's previous crappy javadoc... Will fix.

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**